Demo/Tweakable/TweakingOptionCell.swift

```diff
@@ -0,0 +1,57 @@
+import FinniversKit
+
+class TweakingOptionCell: UITableViewCell {
+    var option: TweakingOption? {
+        didSet {
+            titleLabel.text = option?.title
+            descriptionLabel.text = option?.description
+
+            if descriptionLabel.text == nil {
+                descriptionLabelBottomAnchor?.isActive = false
+                titleLabel.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -.mediumLargeSpacing).isActive = true
+            }
+        }
+    }
+
+    private lazy var titleLabel: Label = {
+        let label = Label(style: .bodyStrong)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    private lazy var descriptionLabel: Label = {
+        let label = Label(style: .detail)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setup()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setup()
+    }
+
+    var descriptionLabelBottomAnchor: NSLayoutConstraint?
```

Comment:
This can be `private`. Should also probably be placed together with the rest of the private properties.